### DISCUSSION TITLE
library 的翻譯

### DISCUSSION THREAD

**Discussion by liying-c** [python/python-docs-zh-tw#702] [Add Term]
### 初步確認

- [X] 我已經確認過 [術語列表](https://github.com/python/python-docs-zh-tw/wiki/%E8%A1%93%E8%AA%9E%E5%88%97%E8%A1%A8) 中沒有相關術語。
- [X] 我已經確認過 [Python 官方術語列表](https://docs.python.org/zh-tw/3/glossary.html) 中沒有相關術語。

### 術語原文

library

### 術語翻譯

函式庫

### 說明

library目前在我們的文件內沒有固定用詞，L10N有，但不在[術語列表](https://github.com/python/python-docs-zh-tw/wiki/%E8%A1%93%E8%AA%9E%E5%88%97%E8%A1%A8)及[Python 官方術語列表](https://docs.python.org/zh-tw/3/glossary.html)列出的範圍內，並且L10N有兩個台灣翻譯，應固定統一翻譯名詞並收錄在repository以利查詢。

搜了一下目前repositroy，library基本上都是使用`函式庫`，故建議統一使用該用詞。

<img width="788" alt="image" src="https://github.com/python/python-docs-zh-tw/assets/67285884/0df88d76-26d3-49c5-8fb3-e20fd844f4cf">

### 參考資料

https://hackmd.io/@l10n-tw/glossaries

**Comment by mattwang44** [comment]
我覺得這個已經算是慣例，不需要再討論，已新增至術語表中，感謝回報！
